analyses/mwi/dryspells_trigger/07_ecmwf_monthly_skill_general.md

```diff
@@ -53,6 +53,10 @@ hdx_blue='#66B0EC'
 
 #### Set config values
 
+```python
+use_incorrect_area_coords=False
```

Comment:
My general "style" is to use all caps in python scripts, indicating that they are global variables. In notebooks, since they are not really run as scripts and there is not a notion of a global variable perse, I normally don't use capse. Does that make sense to you? 

Definitely agree it is something we should have one format for that we define in our coding practices, lets remember when refactoring!